tox.ini

```diff
@@ -103,7 +103,7 @@ commands=
   cp {toxinidir}/infrastructure-playbooks/rolling_update.yml {toxinidir}/rolling_update.yml
   ansible-playbook -vv -i {changedir}/hosts {toxinidir}/rolling_update.yml --extra-vars "\
       ireallymeanit=yes \
-      ceph_stable_release={env:UPDATE_CEPH_STABLE_RELEASE:kraken} \
+      ceph_stable_release={env:UPDATE_CEPH_STABLE_RELEASE:nautilus} \
```

Comment:
nautilus does not exist yet :/ why don't we use `dev`? So we always do from the last stable release to `dev`?

Reply:
the thing is that we will have to change it from 'dev' to 'nautilus' anyway when we will branch the new stable release

Reply:
I know, but this is the upgrade call, so we have to `force` master always, I'm not sure if there is a solution for this.
Or perhaps we should call this `ansible-playbook -vv -i "localhost," -c local {toxinidir}/tests/functional/dev_setup.yml --extra-vars "change_dir={changedir} ceph_dev_branch={env:CEPH_DEV_BRANCH:master} ceph_dev_sha1={env:CEPH_DEV_SHA1:latest}" --tags "vagrant_setup"`

Or maybe it's already called because we run `dev-` scenario?